### PR TITLE
version 1.8.1 Changes:

### PR DESCRIPTION
**Pull request by KavishreeShanmugam11**
1. Modified connect cohesity cluster commandlet to accept Multi-Factor Authentication
2.Modified protection source script, which append cohesity server to the url before calling invoke-rest api method

### CODE REVIEW

File: src/Cohesity.Powershell.Models/AccessTokenCredential.cs [hunk 1]
```diff
@@ -21,38 +21,88 @@ namespace Cohesity.Model
...
-        /// <param name="username">Specifies the login name of the Cohesity user. (required).</param>
-        public AccessTokenCredential(string domain = default(string), string password = default(string), string username = default(string))
+        /// <param name="otpCode">Specifies OTP code for MFA verification..</param>
+        /// <param name="otpType">Specifies OTP type for MFA verification. &#39;Totp&#39; implies the code is TOTP. &#39;Email&#39; implies the code is email OTP..</param>
+        /// <param name="password">Specifies the password of the Cohesity user account..</param>
+        /// <param name="privateKey">Specifies the matching private key of the above certificate..</param>
+        /// <param name="username">Specifies the login name of the Cohesity user..</param>
+        public AccessTokenCredential(string certificate = default(string), string domain = default(string), string otpCode = default(string), OtpTypeEnum? otpType = default(OtpTypeEnum?), string password = default(string), string privateKey = default(string), string username = default(string))
         {
+            this.Certificate = certificate;
```
Comment (pyashish) [inline src/Cohesity.Powershell.Models/AccessTokenCredential.cs:62]: Should we leave certificate/private key out of this ?

Comment (KavishreeShanmugam11) [reply src/Cohesity.Powershell.Models/AccessTokenCredential.cs:62]: I created this model with latest yaml file, to include otpType and otpcode in this model. This includes all other variables as well.

Comment (pyashish) [reply src/Cohesity.Powershell.Models/AccessTokenCredential.cs:62]: Ack, lets keep iit.

Comment (jason-carter-bzy) [reply src/Cohesity.Powershell.Models/AccessTokenCredential.cs:62]: @KavishreeShanmugam11 do you know when 1.8.1 is going for build?  I'm waiting for the fix to Get-CohesityProtectionSource (I have it locally) as I need to push the API through our release pipeline

**Comment by burnziii** [comment]
Any news about release of 1.8.1?
Does it already support 6.8 cluster version?

**Pull request merged**